lib/i18n.js

```diff
@@ -126,18 +130,18 @@
         });
       },
 
-      _scheduleObservers: Ember.on('init', function() {
+      _scheduleObservers: function() {
```

Comment:
The official policy is that addons have prototypes turned off by default actually (starting with 0.2.x?), so this is good advice